apparmor.d/profiles-g-l/hdparm

```diff
@@ -29,6 +29,14 @@ profile hdparm @{exec_path} flags=(complain) {
   # Image files
   owner @{user_img_dirs}/{,**} r,
 
+  # interaction with tlp
+  owner @{sys}/devices/@{pci}/ta@{int}/host@{int}/**/**/power/autosuspend_delay_ms r,
+  owner @{sys}/devices/@{pci}/ta@{int}/host@{int}/scsi_host/host0/link_power_management_policy rw,
+  owner @{sys}/devices/@{pci}/ta@{int}/host@{int}/target*/**/block/{sda,sr0}/* r,
+  owner @{sys}/devices/@{pci}/ta@{int}/host@{int}/target*/**/block/{sda,sr0}/dev r,
+  owner @{sys}/devices/@{pci}/ta@{int}/host@{int}/target*/**/block/sda/sda@{int}/dev r,
+  owner @{sys}/devices/@{pci}/ta@{int}/host@{int}/target*/**/block/sda@{int}/dev r,
+
```

Comment:
Most of this should be covered by disk read.
Note: you cannot have path specific to a given hardware in a rule. Eg: `{sda,sr0}`, `scsi_host`, `host0`